app/scripts/lib/snap-keyring/snap-keyring.ts

```diff
@@ -7,6 +7,19 @@ import type {
 import type { KeyringController } from '@metamask/keyring-controller';
 import { SNAP_MANAGE_ACCOUNTS_CONFIRMATION_TYPES } from '../../../../shared/constants/app';
 import { t } from '../../translate';
+
+/**
+ * Get the addresses of the accounts managed by a given Snap.
+ *
+ * @param controller - Instance of the MetaMask Controller.
+ * @param snapId - Snap ID to get accounts for.
+ * @returns The addresses of the accounts.
+ */
+export const getAccountsBySnapId = async (controller: any, snapId: string) => {
```

Comment:
Can this be typed?

Reply:
I just pushed a commit to address this with better types.